guides/WRITING-DOCUMENTATION.md

```diff
@@ -2,25 +2,79 @@
 
 Documentation for the design system's components are automatically generated from their CSS and JS comments. In order for this to work, you need to follow some conventions set by the tools we use to generate the documentation: [KSS](https://github.com/kss-node/kss-node) and [react-docgen](https://github.com/reactjs/react-docgen).
 
-## File naming
+- [Directory structure](#directory-structure)
+  - [The docs folder structure](#the-docs-folder-structure)
+  - [The src folder structure](#the-src-folder-structure)
+- [Component documentation page formatting](#component-documentation-page-formatting)
+  - [Heading (required)](heading-required)
+  - [Description](#description)
+  - [@Flags](#flags)
+  - [Modifiers](#modifiers)
+  - [Markup](#markup)
+  - [Style guide](#style-guide)
+  - [Guidance](#guidance)
+- [Sample component documentation page](#sample-component-documentation-page)
+- [Documenting React components](#documenting-react-components)
+  - [Description](#description-1)
+  - [PropTypes](#proptypes)
+- [General content page](#general-content-page)
+  - [Front-matter](#front-matter)
+  - [Page content](#page-content)
+
+## Directory structure
 
-Name and structure components as follows in order for docs generation to work as expected.
+```
```

Comment:
Can we remove `child` as the directory structure applies for both core & child ds?

Reply:
I would even be ok with just removing this beginning folder structure. I think it's enough to have a major section that says `Docs folder structure`, we probably dont need to go into the design system package source folder